### DISCUSSION TITLE
Invalid or unexpected token

### DISCUSSION THREAD

**Discussion by GurdipS5** [Zhengqbbb/cz-git#165] [Q&A]
Hi,
I have installed czg, but get this error:

PS D:\Repositories\microK8s\Namespaces\Microk8s.Namespaces> czg break
czg@1.9.1

Invalid or unexpected token

However, if I run czg, the command works and brings up options. I am trying to enable it in an existing repo.

My current package.json is:

{
  "devDependencies": {
    "commitizen": "^4.3.0",
    "czg": "^1.9.1",
    "eslint": "^8.57.0"
  },
  "config": {
    "commitizen": {
      "path": "node_modules/cz-git"
    }
  }
}
 
The "invalid or unexpected token" error implies a syntax error somewhere.

**Comment by GurdipS5** [comment]
I saw a thread on this and the author asked for envInfo, which is below:

PS D:\Repositories\microK8s\Namespaces\Microk8s.Namespaces> npx envinfo --system --binaries

  System:
    OS: Windows 11 10.0.22631
    CPU: (32) x64 13th Gen Intel(R) Core(TM) i9-13900KS
    Memory: 38.78 GB / 63.84 GB
  Binaries:
    Node: 18.17.0 - D:\Program Files\nodejs\node.EXE
    npm: 9.6.7 - D:\Program Files\nodejs\npm.CMD


A new project I create (visual studio) does not have this error with czg, so I think it must be project specific.

**Comment by Zhengqbbb** [comment]
```json
{
  "scripts": {
    "commit": "cz"
  },
  "config": {
    "commitizen": {
      "path": "node_modules/cz-git"
    }
  },
  "devDependencies": {
   "commitizen": "^4.3.0",
    "cz-git": "^1.9.1"
  }
}
```

then try the command like:
```sh
npm run commit
```

**Reply by Zhengqbbb** [reply]
Only the `break` mode command have error or just using `czg` default mode also show

**Reply by Zhengqbbb** [reply]
> A new project I create (visual studio) does not have this error with czg, so I think it must be project specific.

🫣  expect you can find it
